### ISSUE TITLE
Need to capture mode of inheritance from VCI.

### ISSUE THREAD

**Issue by larrybabb** [clingen-data-model/VCI-transformation#2]
@cbizon 
I see that "mode of inheritance" is captured in the VCI data at the top level of the structure.  While I understand the potential for multiple diseases to be included and thus the potential to have different modes of inheritance per condition, I suspect that in the VCI data the MOI will always apply to any of the selected diseases/phenotypes in the group for a given interpretation.  (Please explain if not true)

So, I think it is important to fold the VCI's MOI value into the ClinGen's condition (MendelianCondition) attribute of the same type.  It may need only apply if there is at least one condition specified, maybe it should be dropped or ignored if there is not condition specified in the VCI.

The ClinVar submission uses this MOI value, so it is useful and needed to make the VIC-ClinVar submission work more completely.

Let me know your thoughts.  If you will do this, please indicate an ETA.  Thanks.

**Comment by cbizon** [comment]
Yes, I just reviewed the code and it looks like it's just an oversight that this is not being transformed.   There is a little complexity around the way that the VCI returns this information, because it comes in two fields (the main mode of inheritance, and then a modifier for things like "with maternal imprinting").


This confused me, so I tabled it, but now I need to go back and fix it.  I hope to get this done by the end of the week.?

________________________________

**Comment by cbizon** [comment]
I've pushed an initial version of this.  It should work for most data.  I don't have any real data that has a wide variety of the modeOfInheritance and modeOfInhertianceAdjective valeus, so I don't really think I can consider it fully implemented.  I will leave the ticket open for the moment.
